README.adoc

```diff
@@ -1,71 +1,6 @@
 = FreeBSD Quarterly Status Reports
 
-== Drafts
+This repository has been used to collect and review FreeBSD quarterly status reports from 2017Q4 through 2022Q4.
```

Comment:
> FreeBSD quarterly status reports

instead: 

> reports

(No need to repeat what's in the heading that precedes the sentence).

Reply:
I prefer to keep it here: stressing that the reports were being published on a quarterly basis makes it clearer what 2017Q4 and 2022Q4 mean. I do not know if in the English speaking world such notations are vey common, but in Italian and French I have seen them very rarely: indeed, in those two languages I usually see those kinds of periods refered as trimesters. So I think repeating "quarter"  might help non native English speaker understand better what we are talking about.